lib/rendition.js

```diff
@@ -37,7 +37,7 @@ const Metadata = {
     DC_FORMAT: 'dc:format',
     ENCODING: 'repo:encoding',
     TIFF_IMAGEWIDTH: 'tiff:imageWidth',
-    TIFF_IMAGEHEIGHT: 'tiff:imageHeight'
+    TIFF_IMAGEHEIGHT: 'tiff:imageHeight' // should be TIFF_IMAGELENGTH: 'tiff:imageLength' 
```

Comment:
maybe make it a `// TODO`?

Reply:
If you mean AEM expects it as TIFF_IMAGELENGTH
```suggestion
    TIFF_IMAGEHEIGHT: 'tiff:imageHeight' // AEM expects it as TIFF_IMAGELENGTH: 'tiff:imageLength' 
```

Reply:
any idea why we call it TIFF_* ?

Reply:
We can't change this just now @pheenomenon , because this would pull in AEM testing (which I would like to avoid now, since in the end this is temporary until it's replaced by pipeline). AEM handles it properly already, we should just fix it eventually too.